### PR TITLE
Make beats height dynamic. Calculated based on headset height at song load (fix #4)

### LINKED ISSUE #4: Reports from Quest subreddit of beats being too low.

### PR DESCRIPTION
**Pull request by dmarcos**

### CODE REVIEW

File: src/components/beat.js [hunk 1]
```diff
@@ -145,6 +145,9 @@ AFRAME.registerComponent('beat', {
     this.blockEl.setAttribute('mixin', 'beatBlock');
     this.el.appendChild(this.blockEl);
     this.initMesh();
+
+    this.setBeatsHeight = this.setBeatsHeight.bind(this);
+    this.el.sceneEl.addEventListener('songprocessfinish', this.setBeatsHeight);
```
Comment (dmarcos) [inline src/components/beat.js]: not sure if this is the proper event but seems to work.

File: src/components/beat.js [hunk 2]
```diff
@@ -544,6 +547,16 @@ AFRAME.registerComponent('beat', {
...
     this.returnToPoolTimer -= timeDelta;
     if (this.returnToPoolTimer <= 0) { this.returnToPool(); }
+  },
+
+  setBeatsHeight: function () {
+    var cameraHeight;
+    var cameraWorldPosition = new THREE.Vector3();
+    this.el.sceneEl.camera.localToWorld(cameraWorldPosition);
+    cameraHeight = cameraWorldPosition.y;
+    this.verticalPositions.bottom = cameraHeight * 3 / 8;
```
Comment (dmarcos) [inline src/components/beat.js]: Probably this logic should be part of a system as it's global to all beats.

**Comment by dmarcos** [comment]
Not happy with code but it's a quick fix so people can play the game.

**Comment by ngokevin** [comment]
Pushed to master, using the system, when `isLoading` becomes toggled off, and tweaked to have a minimum height to not go below to ground. https://github.com/supermedium/moonrider/commit/f98b39da1f3d01d49e28c9a09a417f820c84be0c

**Pull request closed without merging**